### PR TITLE
WIP: move talks to a new page #712

### LINKED ISSUE #712: Move talks to a new page
## Summary

As part of re-organizing the Getting Start page, we need to move the '[Talks](https://buildpacks.io/docs/#talks)' section at the top of the page to a new location. This will allow us to add an index of key topics covered in the docs plus an introductory paragraph introducing users to Cloud Native Buildpacks/its capabilities, and use cases.

## What questions should this documentation answer?
- Where to find introduction to Buildpacks videos?
- Where to find Buildpacks conference talks?
- Working Group Meetings Working Group meetings 
- Other videos/talks 

## Who is the target audience for this documentation?

- [x] App Developer
- [x] Buildpack Author
- [x] Operator
- [x] Other:

### PR DESCRIPTION
**Pull request by hyounes4560**
* Adds a new talks page
   * Changes the order of talks by time (Desc)
   * Removes the talks section from the Getting Started landing page
   * Adds an overview paragraph to the Getting Started landing page
   * Updates the level of few section headings in the Getting Started landing page
   * Other minor edits

Closes #712

### CODE REVIEW

**Review by AidanDelaney — COMMENTED** [review]
Looks good so far.  Am happy to approve when it's no-longer a WIP

File: content/docs/talks.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+getting-started=true
++++
+
+## Talks
+
+We love talks to share the latest development updates, covering Buildpacks basics and more, receive feedback and questions, and get to know other members of the community.
+
+### Introductory Talks  
+
+If you are new to the project, you can watch some of introductory talks on [Cloud Native Buildpacks' YouTube channel](https://www.youtube.com/@buildpacks)
```
Comment (natalieparellano) [inline content/docs/talks.md:13]:
```suggestion
```

We have not been very good stewards of our YouTube channel. I would drop the meeting recordings below the conference talks to increase the chances that the first click will be useful.

File: content/docs/talks.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+### Conference Talks
+
+{{< talks >}}
+
+More conference talks are available under the [Conference Talks Playlist](https://www.youtube.com/playlist?list=PL1p8pquzNvRqz3v7Q-OA7wldIxLCNFHcC) on YouTube.
```
Comment (natalieparellano) [inline content/docs/talks.md:19]:
```suggestion
### Conference Talks

Check out some of our most recent and exciting conference talks below. More talks are available in our [Conference Talks Playlist](https://www.youtube.com/playlist?list=PL1p8pquzNvRqz3v7Q-OA7wldIxLCNFHcC) on YouTube.

{{< talks >}}

If you are interested in giving a talk about buildpacks, the linked slides may provide a useful starting point. Please feel free to reach out in Slack if you'd like input or help from the CNB team!
```

It's a bit weird that the `{{< talks >}}` block also drops in a "Resources" with downloadable slides. It makes the "More conference talks are available..." text seem a bit out of place. WDYT?

File: content/docs/talks.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+### Conference Talks
+
+{{< talks >}}
+
+More conference talks are available under the [Conference Talks Playlist](https://www.youtube.com/playlist?list=PL1p8pquzNvRqz3v7Q-OA7wldIxLCNFHcC) on YouTube.  
+
+### Working Group Meetings
+
+Feel free to look through the archive of previous community meetings on our [Working Group YouTube Playlist](https://www.youtube.com/playlist?list=PL1p8pquzNvRpDbbgZ0db0MRA-W5_w0G1U)
```
Comment (natalieparellano) [inline content/docs/talks.md:23]:
```suggestion
### Meetings

Feel free to look through the archive of previous community meetings in our [Working Group Playlist](https://www.youtube.com/playlist?list=PL1p8pquzNvRpDbbgZ0db0MRA-W5_w0G1U) on YouTube. If you would like to attend a Working Group meeting, check out our [community](/docs/community) page.
```

File: data/talks.yml [hunk 4]
```diff
@@ -146,3 +127,22 @@
...
+    experience and security guarantees provided by buildpacks. Cloud-Native Buildpacks (a CNCF sandbox
+    project) represents a generational leap in buildpacks, bringing the power of buildpacks to k8s and
+    beyond.
+  event_name: SpringOne Platform
+  youtube_id: J2SXkmOo8iQ
+  year: 2019
+  resources:
+    - name: Slides
+      icon: file-powerpoint
+      url: https://docs.google.com/presentation/d/14rkLA2hn3nVu_liiW6Y7hy3hSsnz1HueEdLkhvChONM/view
```
Comment (natalieparellano) [inline data/talks.yml:148]:
```suggestion
      url: https://docs.google.com/presentation/d/14rkLA2hn3nVu_liiW6Y7hy3hSsnz1HueEdLkhvChONM/view

```

Comment (natalieparellano) [reply data/talks.yml:148]: I can't help myself, that newline warning just jumps out at me...

File: content/docs/_index.md [hunk 5]
```diff
@@ -19,9 +19,9 @@ summary="Get started with Cloud Native Buildpacks."
```
[the remark this answers is not in the document]
Comment (natalieparellano) [reply content/docs/_index.md:19]: (It used to be Ruby, and it has a new title now...)

**Comment by natalieparellano** [comment]
It looks like DCO is unhappy about 2 commits: https://github.com/buildpacks/docs/pull/713/checks?check_run_id=25260264779

Feel free to squash/force push if that makes it easier to fix the sign-off.

File: content/docs/_index.md [hunk 6]
```diff
@@ -4,9 +4,9 @@ weight=1
 summary="Get started with Cloud Native Buildpacks."
 +++
 
-## Talks
+## Overview
```
[the remark this answers is not in the document]
Comment (hyounes4560) [reply content/docs/_index.md:7]: how about "Welcome to the Cloud Native Buildpacks Documentation"

File: content/docs/_index.md [hunk 7]
```diff
@@ -4,9 +4,9 @@ weight=1
...
 
-{{< talks >}}
+Cloud Native Buildpacks (CNBs) transform your application source code into [container images](https://github.com/opencontainers/image-spec/blob/main/spec.md) that can run on any cloud. With buildpacks, organizations can concentrate the knowledge of container build best practices within a specialized team, instead of having application developers across the organization individually maintain their own Dockerfiles. This makes it easier to know what is inside application images, enforce security and compliance requirements, and perform upgrades with minimal effort and intervention. The CNB project was initiated by Pivotal and Heroku in January 2018 and joined the Cloud Native Computing Foundation (CNCF) as an Apache-2.0 licensed project in October 2018. It is currently an incubating project within the CNCF.
 
 ---
```
[the remark this answers is not in the document]
Comment (hyounes4560) [reply content/docs/_index.md:11]: updated

File: content/docs/_index.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (hyounes4560) [reply content/docs/_index.md:19]: Not sure I understand this

[the remark this answers is not in the document]
Comment (natalieparellano) [reply content/docs/_index.md:19]: Ah, line 16:

```
* [Creating a Cloud Native Buildpack](/docs/for-buildpack-authors/tutorials/basic-buildpack) - Tutorial walking through the creation of a simple Ruby buildpack.
```

Should be:
```
* [How to write a basic buildpack](/docs/for-buildpack-authors/tutorials/basic-buildpack) - Tutorial walking through the creation of a simple nodeJS buildpack.
```

File: content/docs/_index.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (natalieparellano) [reply content/docs/_index.md:7]: I'm fine with this. @AidanDelaney?

File: content/docs/_index.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (hyounes4560) [reply content/docs/_index.md:19]: I don't recall changing this; will update. Thanks

File: content/community.md [hunk 8]
```diff
@@ -69,7 +69,7 @@ If you didn't find anything you'd like to improve while going through the tutori
 
 > Depending our your depth of understanding or desires some components may be more ideal than others.
 
-[talks]: /docs/#talks
+[talks]: /docs/talks
```
[the remark this answers is not in the document]
Comment (hyounes4560) [reply content/community.md:72]: You mean keeping talks on the same page as it used to be? just a different location in the page?

[the remark this answers is not in the document]
Comment (natalieparellano) [reply content/community.md:72]: Ah rather, to put a link to the (new) talks page on the same page where they used to be. I couldn't find any other link to the new page (other than the sidebar).

File: content/docs/_index.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (natalieparellano) [reply content/docs/_index.md:19]: This definitely didn't change with your PR! I just noticed it while doing my review... (we don't have to change it as part of this PR, I just figured we might as well...)

**Comment by hyounes4560** [comment]
@natalieparellano ; FYI, closing this as I created a new one #719 to update the landing page and reorder the talks within the page instead of creating a new page for talks.

**Pull request closed without merging**